docs/source/getting_started.rst

```diff
@@ -146,6 +146,61 @@ You can customize various aspects of the model:
        total_time=10.0
    )
 
+
+Multi-Device Parallelization
+-----------------------------
+
+JCM supports multi-device parallelization using JAX's SPMD (Single Program Multiple Data) sharding. This allows you to split computation across multiple GPUs or TPUs for faster execution, especially useful for higher resolution simulations.
+
+If you don't specify ``spmd_mesh``, JCM runs on a single device by default. This is the recommended approach for smaller resolutions (T31, T42) or when you only have a single GPU/TPU available.
+
+Basic Concepts
+^^^^^^^^^^^^^^
+
+**SPMD Mesh**: Defines how to partition data across devices. The mesh has three dimensions corresponding to ``(x, y, z)`` or ``(longitude, latitude, vertical)``.
+
+**Sharding Strategy**: Typically, you want to shard the longitude dimension first since it usually has the most grid points.
```

Comment:
I think this may depend on the number of vertical layers. For 32 or 64 layers, I found that sharding the dycore in the vertical dimension was typically most effective. In NeuralGCM, we switch back and forth between vertical sharding for the dycore ("dycore sharding") and only sharding horizontally ("physics sharding") when applying physics terms.